Work14/Program.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Work4
```

Comment:
Very long namespace.

Reply:
we did not learn classes yet and I realized within the limits of the material studied, but yes, I agree, it’s not right